packages/calypso-e2e/src/lib/components/notifications-component.ts

```diff
@@ -38,7 +38,8 @@ export class NotificationsComponent {
 	async clickNotificationAction( action: string ): Promise< void > {
 		await this.anchor
 			.getByRole( 'list' )
-			.getByRole( 'button', { name: action, exact: true } )
+			.getByRole( 'button' )
+			.filter( { hasText: action } )
```

Comment:
Filter to find the button that contains the specified action text, instead of an exact match.